touchless/utils/math_utils.py

```diff
@@ -44,3 +44,48 @@ def euclidean(pt1: tuple[float, float], pt2: tuple[float, float]) -> float:
     """
     d: float = math.sqrt((pt1[0] - pt2[0]) ** 2 + (pt1[1] - pt2[1]) ** 2)
     return d
+
+
+def heron_area_by_points(p1: tuple[int, int], p2: tuple[int, int], p3: tuple[int, int], accuracy: int = 3) -> float:
```

Comment:
Not critical, but I found this function is a bit faster: 

```python
def triangle_area(p1: tuple[int, int], p2: tuple[int, int], p3: tuple[int, int], accuracy: int = 3) -> float:
    return round(abs((0.5)*(p1[0]*(p2[1]-p3[1])+p2[0]*(p3[1]-p1[1])+p3[0]*(p1[1]-p2[1]))), accuracy)
```

Source: https://www.cuemath.com/geometry/area-of-triangle-in-coordinate-geometry/

Reply:
Thanks, it's interesting! I'll consider this some later. Yes, it's not critical now. Moreover, "standard" formula is more  more familiar and recognizable.